<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VhallActivityModeChoiceDlg</class>
 <widget class="QDialog" name="VhallActivityModeChoiceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>604</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>604</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>选择直播类型</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>36</number>
   </property>
   <property name="leftMargin">
    <number>29</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>29</number>
   </property>
   <property name="bottomMargin">
    <number>37</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="titlehLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labTitle">
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 12pt &quot;微软雅黑&quot;;
color:#969696;
	background-color: rgb(43, 44, 46);
}</string>
       </property>
       <property name="text">
        <string>选择直播类型</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>433</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <property name="horizontalSpacing">
      <number>30</number>
     </property>
     <property name="verticalSpacing">
      <number>14</number>
     </property>
     <item row="0" column="0">
      <widget class="QFrame" name="frame_live">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>194</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{ 
border: 1px solid #323232;
border-radius: 5px;
}
QFrame:hover{ 
border: 1px solid #8e2928;
border-radius: 5px;
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>24</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_live">
            <property name="minimumSize">
             <size>
              <width>49</width>
              <height>49</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>49</width>
              <height>49</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{ 
border: 0px;
}
QFrame:hover{ 
border: 0px;
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_live_tip">
          <property name="styleSheet">
           <string notr="true">QFrame{ 
border: 0px;
}
QLabel{
color:rgb(70,70,70);
}</string>
          </property>
          <property name="text">
           <string>直播</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>27</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFrame" name="frame_interactivity">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>194</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame#frame_interactivity{ 
border: 1px solid #323232;
border-radius: 5px;
}
QFrame#frame_interactivity:hover{ 
border: 1px solid #8e2928;
border-radius: 5px;
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>201</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_interactivity">
            <property name="minimumSize">
             <size>
              <width>49</width>
              <height>49</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>49</width>
              <height>49</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{ 
border: 0px;
}
QFrame:hover{ 
border: 0px;
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_interactivity_tip">
          <property name="styleSheet">
           <string notr="true">QFrame{ 
border: 0px;
}
QLabel{
color:rgb(70,70,70);
}</string>
          </property>
          <property name="text">
           <string>多人连麦</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>27</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>248</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>248</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#979797;
</string>
       </property>
       <property name="text">
        <string>一对多的直播形式，多种软件演示功能，插播视频功能，支持采集卡</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#979797</string>
       </property>
       <property name="text">
        <string>多对多的直播形式，可实现多嘉宾互动+直播的形式</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../VhallLive/project/VhallLive.qrc"/>
 </resources>
 <connections/>
</ui>
